platform/android/os_android.cpp

```diff
@@ -106,6 +107,9 @@ void OS_Android::initialize_core() {
 		DirAccess::make_default<DirAccessJAndroid>(DirAccess::ACCESS_RESOURCES);
 	DirAccess::make_default<DirAccessUnix>(DirAccess::ACCESS_USERDATA);
 	DirAccess::make_default<DirAccessUnix>(DirAccess::ACCESS_FILESYSTEM);
+#ifndef NO_NETWORK
```

Comment:
Another undocumented define probably introduced for proprietary ports... We'll really have to document all this for 4.0 and review whether they all make sense (and maybe give them more consistent names).

Reply:
Yeah, in the old times we used to be able to compile without network support. I added it there for future reference (importing it from `OS_Unix`)

Reply:
I lack the context, @Faless can you clarify why the `define` is needed?